From b37694a9e374f541b77d0c8a22d54594556048cf Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Mon, 23 Mar 2026 05:05:47 +0000
Subject: [PATCH 36/39] WIP: dt-bindings: arm: amlogic: add support for GTMEDIA
 GTT-2

The GTMEDIA GTT-2 is an Android STB using the Amlogic S905D chip

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index 9f73a0054fb2..ad466660b9b5 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -108,6 +108,7 @@ properties:
           - enum:
               - amlogic,p230
               - amlogic,p231
+              - gtmedia,gtt2
               - libretech,aml-s905d-pc
               - osmc,vero4k-plus
               - phicomm,n1
-- 
2.43.0

